models/database.go

```diff
@@ -687,6 +687,10 @@ var databaseSchema = [][]string{
 		`ALTER TABLE percona_sso_details
 			ADD COLUMN organization_id VARCHAR`,
 	},
+	58: {
+		`UPDATE agents SET mongo_db_tls_options = jsonb_set(mongo_db_tls_options, '{stats_collections}', '[]')
```

Comment:
@BupycHuk In the current form, update will set `stats_collections` field in JSON blob to an empty array. This means that we are losing configuration data for MongoDB exporters.

How about converting updated entry to `to_jsonb(string_to_array(mongo_db_tls_options->>'stats_collections', ',') `instead of `[]`. This should update the configuration JSON blob to work well with newer version of PMM.

Reply:
yeah, that would be much better